package.json

```diff
@@ -8,6 +8,7 @@
     "start-dev": "cross-env NODE_PATH=. node-dev server.js",
     "test": "cross-env NODE_PATH=. NODE_ENV=test nyc mocha 'test/unit/**/*.test.js'",
     "e2e": "cross-env NODE_PATH=. NODE_ENV=test codeceptjs run -c test/e2e/ --steps",
+    "test-unit": "cross-env NODE_ENV=test NODE_PATH=. mocha test/unit --recursive",
     "test-all": "cross-env NODE_PATH=. NODE_ENV=test yarn test && yarn e2e",
```

Comment:
Why do we need both of these `test` and `test-unit` - they essentially do the same thing?